src/components/stats/l2-speaker-stats.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Bar } from 'react-chartjs-2';
+import useSWR from 'swr';
+import { getL2SpeakersStats } from '../../services/stats-api';
+import { setGoal } from '../../store/contribute/actions';
+import { theme } from '../../styles/global';
+
+interface L2SpeakerData {
+    total: number;
+    valid: number;
+    invalid: number;
+}
+
+const options = {
+    scales: {
+        yAxes: [
+            {
+                stacked: false,
+                ticks: {
+                    beginAtZero: true,
+                },
+                scaleLabel: {
+                    display: true,
+                    labelString: 'uttr',
+                },
+            },
+        ],
+    },
+    legend: { display: false },
+};
+
+const L2SpeakerStats: React.FunctionComponent = () => {
+    const { data, error } = useSWR('l2speaker', getL2SpeakersStats);
+
+    const generateChart = () => {
+        const barChartData = generateBarChartData(data);
+        return <Bar data={barChartData} options={options} />;
+    };
+    const generateBarChartData = (indata: L2SpeakerData) => {
+        const labels = ['Total (uttr)', 'Valid (uttr)', 'Invalid (uttr)'];
+        const barData = [indata.total, indata.valid, indata.invalid];
+        const goal = [50000, 0, 0];
```

Comment:
```suggestion
        const goal = [0, 50000, 0];
```

Reply:
I'll keep the goal for collected data as of now. Lets update this when we know how much we need to release. Then we might need to tweak these numbers to exclude data from other releases as well.